build/charts/flow-aggregator/values.yaml

```diff
@@ -104,9 +104,9 @@ flowLogger:
   recordFormat: "CSV"
   # -- Filters can be used to select which flow records to log to file. The provided filters are
   # OR-ed to determine whether a specific flow should be logged. By default, all flows are logged.
+  # With the following filters, only flow which are denied because of a network policy will be logged:
```

Comment:
only flow -> only flows?
network policy -> NetworkPolicy?

Reply:
fixed the first one
for the second one, I don't think we really have a convention when we talk about "network policies" in general (K8s NetworkPolicies, Antrea-native policies) and not about the K8s NetworkPolicy API specifically.